lib_async/ocaml_async_kafka.c

```diff
@@ -214,6 +214,7 @@ CAMLprim value ocaml_kafka_async_consumer_poll(value caml_kafka_handler) {
     option = caml_alloc_small(1, 0);
     Field(option, 0) = ocaml_kafka_async_extract_message(rkm);
     result = OK(option);
+    rd_kafka_message_destroy(rkm);
   }
```

Comment:
Shouldn't the message be destroyed even in case of an error?

```suggestion
  }
  rd_kafka_message_destroy(rkm);
```

Reply:
oops, good catch!